packages/vue/src/components/atoms/SfArrow/SfArrow.html

```diff
@@ -2,7 +2,8 @@
   <button v-on="$listeners" class="sf-arrow__button" type="button">
     <!--@slot Use this slot to replace arrow icon-->
     <slot>
-      <svg class="sf-arrow__icon"viewBox="0 0 24 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <svg class="sf-arrow__icon" viewBox="0 0 24 12" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false">
+        <title>Arrow</title>
```

Comment:
Screen readers do not all the time read <title> by default. See: https://simplyaccessible.com/article/7-solutions-svgs/
The best solution is adding an element next to svg with `visual-hidden` attribute and have `aria-hidden=”true”` on the SVG element to prevent screen reader to read what is not needed to be read.

Reply:
Sounds good, I just updated the PR with the changes, the a11y checks in storybook look much better now:

<img width="485" alt="Screenshot 2019-10-08 at 8 37 56 pm" src="https://user-images.githubusercontent.com/661330/66427396-bf81b780-ea0b-11e9-9ebc-475c1f07b0a3.png">